test/cookbooks/test/recipes/default.rb

```diff
@@ -74,7 +78,10 @@
   on :weighted_resources
   on :converge_start
 
+  method_defined_outside_choregraphie_context
```

Comment:
why adding it in both the root and the before block of the choregraphie?

Reply:
to demonstrate both are working